src/qt/pivx/settings/settingsfaqwidget.h

```diff
@@ -16,13 +18,13 @@ class SettingsFaqWidget : public QDialog
     Q_OBJECT
 
 public:
-    explicit SettingsFaqWidget(QWidget *parent = nullptr);
+    explicit SettingsFaqWidget(PIVXGUI* _window = nullptr, QWidget *parent = nullptr);
```

Comment:
Same as what said in other places, instead of input twice the window object (the `QWidget* parent` is a `PIVXGUI*` already) and stores it on the object, can just refactor the QWidget type to PIVXGUI.

Reply:
done